mitosheet/src/components/taskpanes/UpdateImports/UpdateImportsTaskpane.tsx

```diff
@@ -53,7 +53,9 @@ export interface ReplacingDataframeState {
 
 export interface FailedReplayData {
     analysisName: string,
-    error: MitoError
+    analysis: SavedAnalysis | undefined,
+    error: MitoError,
+    ignoreAuthorHash?: boolean
```

Comment:
I thought that this was only going to be used for testing. Why would we want to ignore the author hash?

Reply:
I'll rename this to be more clear. This should be called `explicitlyTrustAnalysisByIgnoringAuthorHash`